.github/workflows/maven.yml

```diff
@@ -27,4 +27,20 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-jdk-${{ matrix.java }}-mvn-
       - name: Build it with Maven
-        run: mvn clean verify -B -Pqulice
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+             mvn clean verify -B -Pqulice
+          elif [ "$RUNNER_OS" == "Windows" ]; then
+               mvn clean test -B -Pqulice
+          else
+             echo "$RUNNER_OS not supported"
+             exit 1
+          fi
+        shell: bash
+  # @todo #309:30min Run Etcd in windows containers while testing on windows.
```

Comment:
And this todo also is not for CI scripts, please fully revert workflow file and put all related changes to test case only

Reply:
@g4s8 done